File: practice/4/kadai4.js
class Obj {
    constructor(value1,value2) {
        this.key = value1,
        this.str = value2
    }

    getKey() {
        return this.key;
    }

    getStr() {
        return this.str;
    }
}
let obj = new Obj(101,"This is hell.");
console.log(obj.getKey());
console.log(obj.getStr());

File: practice/4/kadai4.py
class Obj:
    def __init__(self,value1,value2):  ##__init__がコンストラクタ
            self.key = value1
            self.str = value2

    def getKey(self):
            return self.key

    def getStr(self):
            return self.str

obj = Obj(101,"This is hell.")
print(obj.getKey())
print(obj.getStr())